.clang-format

```diff
@@ -76,7 +76,7 @@ IncludeCategories:
     Priority:        3
 IncludeIsMainRegex: '([-_](test|unittest))?$'
 IndentCaseLabels: true
-IndentPPDirectives: AfterHash
+IndentPPDirectives: None
```

Comment:
I don't think our code is consistent about this, but "None" can be a bit hard to read. I don't think we do AfterHash anywhere, but it does look interesting. BeforeHash is probably closest to what we do now.

Reply:
I'll change it to BeforeHash. I generally think it's more legible. I think it's far more important to just pick a consistent indentation and stick to it.